ci/azure-test-stable.yml

```diff
@@ -25,14 +25,6 @@ jobs:
   - template: azure-is-release.yml
 
   - ${{ each crate in parameters.crates }}:
-    # Run with default crate features
-    - script: cargo test
-      env:
-        LOOM_MAX_PREEMPTIONS: 2
-        CI: 'True'
-      displayName: ${{ crate }} - cargo test
-      workingDirectory: $(Build.SourcesDirectory)/${{ crate }}
-
```

Comment:
This is removed because we don't care about loom if no features are enabled (because there's nothing to test)?

Reply:
This doesn’t run loom. Loom is separate. The tests don’t run without setting feature flags